modules/ROOT/pages/stats-monitoring.adoc

```diff
@@ -117,113 +183,130 @@ The response is a JSON object with the following schema.
+See: {url-golang-memstats} for more on memory allocator statistics
 
+==== admin_net_bytes_recv
+Description::
+Total number of bytes received (since node start-up) on the network interface to which the Sync Gateway xref:config-properties.adoc#adminInterface[admin interface] is bound.
++
+By default, that is the number of bytes received on `127.0.0.1:4985` since node start-up.
+Use Case::
 Throughput on admin interface = `admin_net_bytes_recv` / `admin_net_bytes_sent`
 
-===== admin_net_bytes_sent
-
-Total number of bytes sent on the network interface that the Sync Gateway xref:config-properties.adoc#adminInterface[admin interface] is bound to, since start-up.
+==== admin_net_bytes_sent
+Description::
+Total number of bytes sent (since node start-up) on the network interface to which the Sync Gateway xref:config-properties.adoc#adminInterface[admin interface] is bound.
++
+By default, that is the number of bytes sent on `127.0.0.1:4985` since node start-up.
 
-By default, that is the number of bytes sent on `127.0.0.1` since start-up.
+Use Case::
+This metric can be useful in calculating the throughput on Sync Gateway's admin interface:
 
-Throughput on admin interface = `admin_net_bytes_recv` / `admin_net_bytes_sent`
+_Throughput on admin interface_ = `admin_net_bytes_recv` / `admin_net_bytes_sent`
 
-===== error_count
+==== error_count
 
 Number of errors logged.
 
+==== memory allocator statistics
+
+See: {url-golang-memstats} for more on memory allocator statistics
+
 ===== go_memstats_heapalloc
 
-Go memstats.HeapAlloc
+Go memstats.HeapAlloc -- {goto-url-golang-memstats}
 
 ===== go_memstats_heapidle
 
-Go memstats.HeapIdle
+Go memstats.HeapIdle -- {goto-url-golang-memstats}
 
 ===== go_memstats_heapinuse
 
-Go memstats.HeapInuse
+Go memstats.HeapInuse -- {goto-url-golang-memstats}
 
 ===== go_memstats_heapreleased
 
-Go memstats.HeapReleased
+Go memstats.HeapReleased -- {goto-url-golang-memstats}
 
 ===== go_memstats_pausetotalns
 
-Go memstats.PauseTotalNs
+Go memstats.PauseTotalNs -- {goto-url-golang-memstats}
 
 ===== go_memstats_stackinuse
 
-Go memstats.StackInuse
+Go memstats.StackInuse -- {goto-url-golang-memstats}
 
 ===== go_memstats_stacksys
 
-Go memstats.StackSys
+Go memstats.StackSys -- {goto-url-golang-memstats}
 
 ===== go_memstats_sys
 
-Go memstats.Sys
+Go memstats.Sys -- {goto-url-golang-memstats}
 
-===== goroutines_high_watermark
+==== goroutines_high_watermark
 
 Peak number of go routines since process start.
 
-===== num_goroutines
+==== num_goroutines
 
 Number of goroutines.
 
-===== process_cpu_percent_utilization
+==== process_cpu_percent_utilization
 
-CPU utilization (%).
+Description::
+The CPU's utilization as percentage value.
+Derivation::
+The CPU usage calculation is performed based on _user_ and _system_ CPU time, but it doesn't include components such as `iowait`.
+This means that the values of `process_cpu_percent_utilization` and `%Cpu`, returned when running the `top` command, will *differ*  .
```

Comment:
Delete space after `*differ*` and before full stop.

Reply:
Change